python/tvm/auto_scheduler/relay_integration.py

```diff
@@ -171,7 +171,7 @@ def extract_tasks(
                 desc=",".join(func_names),
             )
         )
-        weights.append(weight)
+        weights.append(int(weight))
```

Comment:
Need to confirm that nothing expects this to be an `IntImm` downstream

Reply:
Ah, I see that `_compute_score` recently had some changes to ensure that it returns a float (02f885aacde5739c060507afa0b534019d5439cc) which address this.